React/Modules/RCTTiming.m

```diff
@@ -133,6 +138,14 @@ - (dispatch_queue_t)methodQueue
   return RCTJSThread;
 }
 
+- (NSDictionary *)constantsToExport
```

Comment:
Please define this just on the JS side. This will require the module to be created on the main thread just to send some static numbers over.